elements/bulbs-pdfobject/bulbs-pdfobject.js

```diff
@@ -21,7 +21,7 @@ class BulbsPDFObject extends BulbsHTMLElement {
   embedPDF () {
     let poster = this.getAttribute('poster');
     let options = {
-      fallbackLink: `<a href='[url]'><div class="fallback-btn">Tap to explore document</div><img src="${poster}"></a>`,
+      fallbackLink: `<a href='[url]' data-track-category="SF: Trump Docs 2017" data-track-action="Embed Fallback" data-track-label='[url]'><div class="fallback-btn">Tap to explore document</div><img src="${poster}"></a>`,
```

Comment:
There's no way to pass in `SF: Trump Docs 2017` from the integrating app template?  Don't like hardcoding that